### DISCUSSION TITLE
Vs Guest

### DISCUSSION THREAD

**Discussion by Kaique62** [luckydog7/Funkin-android#1481] [mods]
Preview: https://youtu.be/GQnjIhBBRuM
Download:https://www.mediafire.com/file/te38jwhvzkm4x0l/Vs_Guest.apk/file
Screenshots:
![Screenshot_20210628-173321](https://user-images.githubusercontent.com/81991014/123700575-08f4e480-d837-11eb-946e-44a7c2db4579.png)
![Screenshot_20210628-173428](https://user-images.githubusercontent.com/81991014/123700644-2033d200-d837-11eb-90f4-4666c33d10f2.png)
![Screenshot_20210628-173417](https://user-images.githubusercontent.com/81991014/123700817-58d3ab80-d837-11eb-8352-f5c4cfadae76.png)

**Comment by DzGithub2805** [comment]
How you add week?

**Reply by Kaique62** [reply]
I code it using a pc

**Reply by DzGithub2805** [reply]
I knew it
